internal/test/wmcb/wmcb_test.go

```diff
@@ -144,7 +160,13 @@ func (vm *wmcbVM) runTestBootstrapper(t *testing.T) {
 	err := vm.initializeTestBootstrapperFiles()
 	require.NoError(t, err, "error initializing files required for TestBootstrapper")
 
-	err = vm.runTest(e2eExecutable + " --test.run TestBootstrapper --test.v --platform-type=aws")
+	if !*dockerRuntime {
```

Comment:
After reading the proposed changes, I feel the flag name should be `--containerdRuntime`. It will lean the execution flow towards the positive side, instead of always negating the proposed flag.

I know it's late for this comment 😞

Reply:
Agree. It was just --Runtime and it was easy to read the code flow. Then we changed it to bool and still wanted to give preference to docker runtime and use contained only for testing and community operator. Hence all the confusion :)